testing/run_tests.py

```diff
@@ -282,13 +284,47 @@ def EnsureJavaTestsAreBuilt(android_out_dir):
   RunCmd(gn_command, cwd=buildroot_dir)
   RunCmd(ninja_command, cwd=buildroot_dir)
 
+def EnsureIosTestsAreBuilt(ios_out_dir):
```

Comment:
I know we aren't doing this in this file but a small docstring would be helpful here.  This file is growing and getting harder to maintain.  (I'm sure you know but you put docstrings inside python functions)

Reply:
ya for sure. Added